src/app/tabs/newAdmin/addresses/operations.js

```diff
@@ -103,6 +122,57 @@ const setMultiChainAddress = (domain, chainId, address, isNew) => async (dispatc
   );
 };
 
+/**
+ * Sets values in the Definitive Resolver.
+ * @param {string} domain the domain to be set
+ * @param {uint8} chainId chain Id for the coin
+ * @param {address} address address to be set
+ * @param {bool} isNew is this a new address?
+ */
+const setDefinitiveAddress = (domain, chainId, address, isNew) => async (dispatch) => {
+  const chainName = getChainNameById(chainId);
+  const chainIndex = getIndexById(chainId);
+  const accounts = await window.ethereum.enable();
+  const currentAddress = accounts[0];
+
+  dispatch(requestSetChainAddress(chainName));
```

Comment:
Changed it to this:
```
const chainName = getChainNameById(chainId);
dispatch(requestSetChainAddress(chainName))

const chainIndex = getIndexById(chainId);
...
```
Since I need the chainName to send to redux.

Reply:
True. I wrote it on the fly. Good catch, thanks.